cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.md

```diff
@@ -0,0 +1,44 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024. The changes were deployed first in the [CERN Open Data QA instance](https://opendata-qa.cern.ch), and then propagated to the production instance.
+
+The list of changes include:
+
+* Change of technology, from Angular to ReactJS and Bootstrap[3,4] to Semantic UI for the web interface
```

Comment:
- User interface was changed from Angular to React which improved rendering on mobile devices.  

Basically, it's good to always end the user-oriented release news with a little "so what?" message explaining to the target audience why this may be interesting for them.

Reply:
ok, done